### PR TITLE
Fix for #417

### PR DESCRIPTION
**Pull request by ramdesh**
- [x] Change 'incident mode of receipt' to 'mode of receipt'
- [x] Remove 'responsible political party' and change this into "Against who"
- [x] Change order (switch) police station and division

### CODE REVIEW

File: frontend/src/incident-filing/components/IncidentFormInternal.js [hunk 1]
```diff
@@ -623,7 +623,7 @@ function IncidentFormInternal(props) {
                                         </Grid>
                                         <Grid item xs={12} sm={6}>
                                             <FormControl className={classes.formControl}>
-                                                <InputLabel htmlFor="politicalParty">Reponsible Political Party</InputLabel>
+                                                <InputLabel htmlFor="politicalParty">Against who</InputLabel>
```
Comment (MrClemRkz) [inline frontend/src/incident-filing/components/IncidentFormInternal.js]: Let's remove this select component completely.

Comment (ramdesh) [reply frontend/src/incident-filing/components/IncidentFormInternal.js]: Removed.

Comment (MrClemRkz) [reply frontend/src/incident-filing/components/IncidentFormInternal.js]: awesome.

**Comment by ramdesh** [comment]
Merging in to `dev` branch

**Pull request merged**